### ISSUE TITLE
Clarify why search blacklist exists

### ISSUE THREAD

**Issue by appinteractive** [lbryio/odysee-ios#79]
All Covid related terms are blocked/blacklisted through the iOS app but not the web app. This leaves a shady censoring feeling and us even worst then YouTube. I would wish some transparency in that topic. 

Blocked terms are: 
- Covid
- Corona
- SARS 

![image](https://user-images.githubusercontent.com/432828/113550589-95568900-95f3-11eb-9f8e-d9417e9a1a47.png)

**Comment by appinteractive** [comment]
Found the blacklist but without explanation:
```
    static let keywordsForEmptyResults = [
        "corona",
        "coronavirus",
        "corona virus",
        "sars-cov-2",
        "sars cov 2",
        "sarscov2",
        "sars",
        "covid",
        "covid-19",
        "covid19",
        "covid 19"
    ]

```

https://github.com/lbryio/odysee-ios/blob/d782e68f02da6a8e7dfe300dd2321d65d8fc335d/Odysee/Utils/Lighthouse.swift#L15

**Comment by appinteractive** [comment]
Even the commit message does not mention anything about that change. @akinwale why? What is the reasoning behind that?

**Comment by kauffj** [comment]
@appinteractive we would never do this of our own initiative, this was a way of complying with iOS policy around this type of content.

We will prioritize making a chance to be clear about why this behavior exists.

**Comment by appinteractive** [comment]
Thanks for the response, could you also point to the paragraph ￼in the AppStore terms that prohibits that kind of content? As far as I can tell no other app does that.

**Comment by kauffj** [comment]
Possible text: "This search term is disabled to comply with iOS content guidelines. The results of this search can be seen on odysee.com" where odysee.com is a link to the execution of the search.

**Comment by kauffj** [comment]
For @appinteractive, we received this as a reply from Apple when submitting our app without this filter:

> We found in our review that your app provides services or requires sensitive user information related to the COVID-19 pandemic. Since the COVID-19 pandemic is a public health crisis, services and information related to it are considered to be part of the healthcare industry. In addition, the seller and company names associated with your app are not from a recognized institution, such as a governmental entity, hospital, insurance company, non-governmental organization, or university.
Per section 5.1.1 (ix) of the App Store Review Guidelines, apps that provide services or collect sensitive user information in highly-regulated fields, such as healthcare, should be submitted by a legal entity that provides these services, and not by an individual developer.

**Comment by appinteractive** [comment]
@kauffj Thank you very much for sharing that information! The question remaining is: why other platforms do not need to filter their search terms? (or at least not all or not by that terms) 

May be there another way of complying to the rules (which are very vague by the way) without using a blacklist? 

Sorry for taking your time on this but that topic or it's treatment just feels wrong to me.

**Comment by akinwale** [comment]
Other platforms do not need to filter their search terms because they are not controlled by Apple. In order to get listed on the App Store, we had to comply with their content guidelines. To do otherwise would result in updates being rejected, or even get the app probably delisted if we try to add features that were initially rejected back in.

I have updated the message displayed for disabled search terms.
